contracts/programs/gummyroll/src/lib.rs

```diff
@@ -6,14 +6,14 @@ use anchor_lang::{
 use borsh::{BorshDeserialize, BorshSerialize};
 use std::mem::size_of;
 
+pub mod error;
 pub mod state;
 pub mod utils;
 
-use crate::state::{
-    merkle_roll::{MerkleRoll, MerkleRollHeader},
-    node::Node,
-};
+use crate::error::GummyrollError;
+use crate::state::{change_log::ChangeLogEvent, merkle_roll::MerkleRollHeader, node::Node};
 use crate::utils::ZeroCopy;
+use concurrent_merkle_tree::{merkle_roll::MerkleRoll, state::Node as TreeNode};
```

Comment:
Do we still need both node objects? It feels pretty confusing

Reply:
Fixed